core/bentley/src/Tracing.ts

```diff
@@ -82,6 +85,7 @@ function flattenObject(obj: object): SpanAttributes {
 /**
  * Enables OpenTelemetry tracing in addition to traditional logging.
  * @public
+ * @deprecated in 4.4 - OpenTelemetry Tracing helpers will become internal in a future release. Apps should use `@opentelemetry/api` directly.
```

Comment:
Document deprecations in NextVersion.md.

Reply:
done